packages/protocol/migrationsConfig.js

```diff
@@ -69,7 +69,7 @@ const DefaultConfig = {
 }
 
 const linkedLibraries = {
-  LinkedList: ['AddressLinkedList', 'SortedLinkedList'],
+  LinkedList: ['AddressLinkedList', 'SortedLinkedList', 'LinkedListTest'],
```

Comment:
I don't think it will have any impact in a production migration (because the test contracts should not be deployed then). Anyway, you can check from this branch what linking inside the testsuite  would look like: https://github.com/mrsmkl/celo-monorepo/compare/linked-list...mrsmkl:linking?expand=1
So one problem is that there is no link property for contracts in Truffle typings. But I can do the same thing for other test contracts too.